.github/workflows/pull-request.yml

```
@@ -348,3 +348,50 @@ jobs:
+          echo "Waiting for the GraphQL endpoint to become available..."
+          for i in {1..30}; do
+            # Check if container exists first
+            if ! docker ps | grep -q talawa-api-1; then
+              echo "Container talawa-api-1 not found. Waiting..."
+              sleep 2
+              continue
+            fi
+            
+            # Make sure curl is installed
+            docker exec talawa-api-1 which curl >/dev/null 2>&1 || docker exec talawa-api-1 apt-get update && docker exec talawa-api-1 apt-get install -y curl >/dev/null 2>&1
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)